components/gene-panels/ui/src/main/resources/PhenoTips/PanelsDisplay.xml

```diff
@@ -484,6 +504,31 @@
 
 #genePanelsPagination {
   font-size: .85em;
+  float: right;
+}
+
+#genePanelsPaginationWrapper {
+  display: block;
+  margin-top: 1em;
+}
+
+#genePanelsDownloadWrapper {
+  float: left;
+}
+
+#genePanelsExcludeGenesWrapper {
+  float: right;
+  font-size: .7em;
+  text-transform: capitalize;
+  margin-right: -20px;
+  color: #4D5860;
```

Comment:
We're trying to avoid hardcoded colors. Please use one from the active color theme, e.g. `$theme.textColor` instead (available thanks to the inclusion of the  `colorThemeInit.vm` template at the beginning of tis CSS block).